src/components/form/HdCheckbox.vue

```diff
@@ -150,7 +150,8 @@ export default {
       this.$emit('input', !this.isChecked);
       this.$nextTick(() => {
         this.validate();
-        this.$refs[this.name].focus();
+        // eslint-disable-next-line no-unused-expressions
+        this.$refs[this.name]?.focus();
```

Comment:
Optional chaining requires a babel plugin, which we're not sure all the projects have.
Also, if this change is not related to the icons, it's better to do it in a separate repo.

Reply:
Aaah bummer 😕

Reply:
Done, we have a separate PR 👍